packages/webpack/test/worker/bundle.js

```diff
@@ -0,0 +1 @@
+(()=>{"use strict";console.log("hmm")})();
```

Comment:
Do we want to check the bundle in?

Reply:
Not sure here. I am open to any suggestion that would let us assert that the rule can bundle the code correctly.

Reply:
We would make a test which consumes the output of the rule and makes some assertion on it

```
const {readFileSync} = require('fs');

const bundlePath = process.argv[3];

const got = readFileSync(bundlePath);
const want = `(()=>{"use strict";console.log("hmm")})();`
if (got !== want) {
  throw new Error("Bundle is incorrect");
}
```

Reply:
But really, just having the build target be present is likely already sufficient. We can handle more elaborate testing in the `examples/react_webpack` example in a followup

Reply:
I see that even if we write a test like above we would be inlining the bundle into the test. so I propose that we keep it the way it is. I agree on elaborate testing with an integration test in examples.